sources/packages/backend/apps/api/src/route-controllers/config/_tests_/e2e/config.controller.getConfig.e2e-spec.ts

```diff
@@ -51,6 +52,7 @@ describe("ConfigController(e2e)-getConfig", () => {
           },
           externalSiteMinderLogoutUrl: fakeEnvVariables.SITE_MINDER_LOGOUT_URL,
         },
+        isFulltimeAllowed: fakeEnvVariables.IS_FULLTIME_ALLOWED === "true",
```

Comment:
I am not following the reason why the `=== "true"` why required here. Can you please elaborate on that?

Reply:
`IS_FULLTIME_ALLOWED: false` is a string in `process.env`. But when retrieved from the `/config` api call, it is a boolean. Therefore the comparison.